tensorbay/cli/utility.py

```diff
@@ -46,7 +47,7 @@ def get_config_filepath() -> str:
     return os.path.join(os.environ[home], ".gasconfig")
 
 
-def read_config(config_filepath: str, profile_name: str) -> Tuple[str, str]:
+def read_profile(config_filepath: str, profile_name: str) -> Tuple[str, str]:
```

Comment:
make `read_profile` private?

Reply:
can we remove the `config_filepath` input, get the config file inside the function?